native/swift/Sources/wordpress-api/WordPressLoginClient.swift

```diff
@@ -25,21 +25,22 @@ public final class WordPressLoginClient {
         self.client = UniffiWpLoginClient(requestExecutor: requestExecutor, middlewarePipeline: middleware)
     }
 
+    /// Uses the proposed site URL to scan the website and return login related information about the site.
+    ///
+    public func details(
+        ofSite proposedSiteUrl: String
+    ) async throws -> AutoDiscoveryAttemptSuccess {
```

Comment:
This should probably just return `client.apiDiscovery(siteUrl: proposedSiteUrl).apiDetails`

IINM there's no other useful information in `AutoDiscoveryAttemptSuccess` anyway

Reply:
`api_root_url` is in `AutoDiscoveryAttemptSuccess` though: https://github.com/Automattic/wordpress-rs/blob/cb6c2c17a973ba9cb121937e0e7b035af6f704a9/wp_api/src/login/url_discovery.rs#L359-L363

Reply:
Ah you're right. Never mind then :)